cli/src/main/java/org/jboss/as/cli/handlers/OperationRequestHandler.java

```diff
@@ -89,7 +89,7 @@ public void handle(CommandContext ctx) throws CommandLineException {
         try {
             final ModelNode result = ctx.execute(op, "Operation request");
             if (Util.isSuccess(result)) {
-                ctx.printLine(result.toString());
+                ctx.printDMR(result);
             } else {
                 throw new CommandLineException(result.toString());
```

Comment:
I commented on https://developer.jboss.org/wiki/CLI-OutputJSON re: exception handling (i.e. I think this is right.)